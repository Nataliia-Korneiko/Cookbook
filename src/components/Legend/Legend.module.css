.list {
  padding: 0 10px;
  display: flex;
  justify-content: space-between;
  width: 300px;
  height: 44px;
  border: none;
  border-radius: 5px;
  background-color: rgb(255, 255, 255);
  font-size: 18px;
  font-weight: 400;
  color: #171718;
}

.item {
  display: flex;
  align-items: center;
}

@media (max-width: 980px) {
  .list {
    display: none;
  }
}
