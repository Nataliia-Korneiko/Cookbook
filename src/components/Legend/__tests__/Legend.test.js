/* eslint-disable react/jsx-props-no-spreading */
import React from 'react';
import { shallow } from 'enzyme';
import Legend from '../Legend';

describe('Legend component', () => {
  const props = {
    items: [
      { level: 'low', color: '#4caf50' },
      { level: 'normal', color: '#2196f3' },
      { level: 'high', color: '#f44336' },
    ],
  };

  const newLegend = shallow(<Legend {...props} />);

  it('renders a properly', () => {
    expect(newLegend).toMatchSnapshot();
  });

  it('renders one ul', () => {
    expect(newLegend.find('ul.recipes__list')).toHaveLength(1);
  });

  it('renders three li', () => {
    expect(newLegend.find('li.recipes__item')).toHaveLength(3);
  });

  it('renders three i', () => {
    expect(newLegend.find('i.material-icons')).toHaveLength(3);
  });
});
