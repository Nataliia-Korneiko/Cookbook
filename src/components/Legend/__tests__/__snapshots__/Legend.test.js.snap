// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Legend renders a properly 1`] = `ShallowWrapper {}`;
